sample/CosmosDBTrigger-CSharp/function.json

```diff
@@ -1,11 +1,11 @@
 ﻿{
     "bindings": [
-      {
-        "type": "cosmosDBTrigger",
-        "name": "input",
-        "direction": "in",
-        "databaseName": "myDatabase",
-        "collectionName": "myCollection"
-      }
+        {
+            "type": "cosmosDBTrigger",
+            "name": "input",
+            "direction": "in",
+            "databaseName": "ItemDb",
```

Comment:
Sample was failing because these were not present. Changing them to be the same as tests use so sample host can start w/o error